### PR TITLE
Ngray/display all categories

### PR DESCRIPTION
**Pull request by ngray122**

### CODE REVIEW

**Comment by ngray122** [comment]
@jaygraygray can you review pr?  2 minor issues with endpoints. I'll create those issues to fix later.

File: client/src/components/card/Category.jsx [hunk 1]
```diff
@@ -30,15 +30,41 @@ const Category = () => {
   }, [header]);
 
   const getColumnsForRow = () => {
-    let items = headers.map((header, index) => {
+    let items = headers.map((header) => {
```
Comment (jaygraygray) [inline client/src/components/card/Category.jsx:33]: I would destructure the variables from `header`, so you don't have to access everything like `header.image` or `header.id` each time.

```suggestion
    let items = headers.map(({ id, image, name, episode, season, price }) => {
```

File: client/src/components/card/Category.jsx [hunk 2]
```diff
@@ -30,15 +30,41 @@ const Category = () => {
...
+                />
+              ) : (
+                ""
+              )}
+              {header.name ? (
+                <CardTitle tag="h4"> {header.name}</CardTitle>
+              ) : (
+                ""
+              )}
+              {/* && header.episode.length < 5 */}
```
Comment (jaygraygray) [inline client/src/components/card/Category.jsx:53]: Do you still need this?

File: client/src/components/card/Category.jsx [hunk 3]
```diff
@@ -30,15 +30,41 @@ const Category = () => {
...
+    let items = headers.map((header) => {
       return (
-        <Col>
-          <Card key={header}>
+        <Col key={header.id}>
+          <OneCard>
             <CardBody>
-              <CardTitle>Name: {header.name}</CardTitle>
-              <CardText>Voiced By: {header.voicedBy}</CardText>
+              {header.image ? (
```
[the remark this answers is not in the document]
Comment (ngray122) [reply client/src/components/card/Category.jsx]: @jaygraygray Can you review my changes?

**Pull request merged**